include/abb_librws/v1_0/rw/retcode.h

```diff
@@ -0,0 +1,44 @@
+#include <abb_librws/rws.h>
+#include <abb_librws/v1_0/rws_client.h>
+
+#include <string>
+
+
+namespace abb :: rws :: v1_0 :: rw
+{
+    using namespace rws::rw;
+}
+
+
+namespace abb :: rws :: v1_0 :: rw :: retcode
+{
+    /**
+     * \brief A structure for containing response of recode resource.
+     */
+    struct RetcodeInfo
+    {
+        int code;                   /** \brief The error code or success code as number */
+        std::string name;           /** \brief The error code or success code as string */
```

Comment:
Is `name` just a string representation of `code`, like `"`123`", or something else? Please change the comment s.t. it is clear.

Reply:
```suggestion
        std::string name;           /** \brief The name of error code or success code */
```